### PR TITLE
Create boardreader.com.xml

### PR DESCRIPTION
**Pull request by TotalCaesar659**

### CODE REVIEW

File: src/chrome/content/rules/boardreader.com.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+		clients.boardreader.com
+		dev01.boardreader.com
+		forum.boardreader.com
+		spider5.boardreader.com
+-->
+<ruleset name="boardreader.com">
+	<target host="boardreader.com" />
+	<target host="www.boardreader.com" />
+	<target host="api.boardreader.com" />
+	<target host="apidemo.boardreader.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/boardreader.com.xml]: `devtools.boardreader.com`

Comment (J0WI) [reply src/chrome/content/rules/boardreader.com.xml]: I didn't notice the cert chain issue, sorry

Comment (TotalCaesar659) [reply src/chrome/content/rules/boardreader.com.xml]: Remove it again?

Comment (J0WI) [reply src/chrome/content/rules/boardreader.com.xml]: I'd rather move it to the comments

Comment (TotalCaesar659) [reply src/chrome/content/rules/boardreader.com.xml]: Added to comments.

Comment (J0WI) [reply src/chrome/content/rules/boardreader.com.xml]: Thanks!

**Pull request merged**